### PR TITLE
Feat/kwaai/scale demo

### PR DESCRIPTION
**Pull request by esanchez411**
# Description
This PR introduces a new feature to the project - a postgres database as a source to RAG and the posibility to visualize the vector database created when adding a new inbox emails, also the rag chat is avaiable to prompt questions about the user emails

# Changes made

- Added a new model "Inbox Email"
- A function to add source from postgres DB
- Modified celery config in docker compose
- Added missing env variables in docker compose (Celery-beat and celery containers)

### CODE REVIEW

File: app/job/tasks.py [hunk 1]
```diff
@@ -2,57 +2,70 @@
...
 
-
 @shared_task
 def process_unseen_emails():
     try:
-        unseen_emails = get_unseen_emails()        
-
+        path = 'utilities/'
         logging.info(f"Directory: {os.popen('ls -lha /app/utilities/').read()}")
+
```
Comment (AnniePachecoJ) [inline app/job/tasks.py]: Please remove useless comments

File: app/job/tasks.py [hunk 2]
```diff
@@ -2,57 +2,70 @@
...
-        else:
-            path = 'utilities/'
-            csv_files = [f for f in os.listdir(path) if f.startswith('inbox_') and f.endswith('.csv')]
-            logging.info("csv_files list:", csv_files)
-            if len(csv_files) > 0:
-                add_rag_source(path + csv_files[-1])
-            for unseen_email in unseen_emails:                
-                context = get_rag_response("Contextualize this email:"+ unseen_email['Body'])
+        else:            
+            # csv_files = [f for f in os.listdir(path) if f.startswith('inbox_') and f.endswith('.csv')]
```
Comment (AnniePachecoJ) [inline app/job/tasks.py]: same remove useless comments

File: app/job/tasks.py [hunk 3]
```diff
@@ -2,57 +2,70 @@
...
-                                             + "\n\nFocus solely on responding within the context and requirements of this specific email thread '"+ unseen_email['Message-ID'] +"'."
-                                             )
+                    id=unseen_email['Id'],
+                    message_id=unseen_email['Message-ID'],
+                    to_address=unseen_email['From'],
+                    subject=unseen_email['Subject'], 
+                    prompt=get_rag_response(
+                        f"Draft a response considering the following guidelines and information:"
+                        f"\n\n1. Address the sender personally, using the name {unseen_email['From']}."
+                        # f"\n\n2. Apply the provided stylistic guidelines: {style}."
```
Comment (AnniePachecoJ) [inline app/job/tasks.py]: Comments

**Review by AnniePachecoJ — CHANGES_REQUESTED** [review]
Please remove useless comments/ loggings

File: app/job/tasks.py (hunk 1 above)
Comment (esanchez411) [reply app/job/tasks.py]: Solved

File: app/job/tasks.py (hunk 2 above)
Comment (esanchez411) [reply app/job/tasks.py]: Solved

File: app/job/tasks.py (hunk 3 above)
Comment (esanchez411) [reply app/job/tasks.py]: Solved

**Pull request merged**